application/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: "3.4"
+
+services:
+  server_a:
+    build:
+      context: ..
+      target: development
+    working_dir: /app/application
+    command: "true"
+    volumes:
+      - .:/app/application
+      - ~/gcp-service-credentials/priotest-a-prod-dc20730c9063.json:/app/.credentials
+    environment:
+      - DATA_CONFIG=/app/application/config/content.json
+      - SERVER_ID=A
+      - SHARED_SECRET=m/AqDal/ZSA9597GwMM+VA==
+      - PRIVATE_KEY_HEX=624BFDF22F729BBFD762B3D61930B876F3711B200A10F620FEAC6FD792A2BD08
+      - PUBLIC_KEY_HEX_INTERNAL=AB0008BDE17581D3C45CA8CEACB3F7CE6FB48FEF98AA78597A6955633F54D628
+      - PUBLIC_KEY_HEX_EXTERNAL=68499CBDCAE6B06CAC0C86D255A609B6AFF66A56087803CFE4BD998C7E20220C
+      - BUCKET_INTERNAL_PRIVATE=project-a-private
+      - BUCKET_INTERNAL_SHARED=project-a-shared
+      - BUCKET_EXTERNAL_SHARED=project-b-shared
+      - GCP_CREDENTIALS=/app/.credentials
```

Comment:
@haroldwoo Is this a realistic method for deploying the prio processor? I'm mounting the google application credentials into the docker container like they are in this integration tests for Prio? One of the first steps is to call an authentication function that runs `gcloud auth activate-service-account --key-file "${GCP_CREDENTIALS}"`.

Reply:
Should be okay as long as this is this a private repository.

Reply:
There aren't any private repositories involved other than the `cloudops-infra`. Mounting the json key file is the easiest way to test that the container works, but I'm not sure how it'll work when running the container in the kubernetes cluster.

Reply:
k8s clusters would typically use secrets for this sort of thing.  https://kubernetes.io/docs/concepts/configuration/secret/
If this is a long lived cluster we can add them manually

Reply:
also, what are the credentials used for? We may be able to grant the svc account role assigned to the gke cluster the right permissions so the container workload uses them.

Reply:
It's being using for `gsutil`, in particular `stat`, `rsync`, and `cp`. If there's a simpler way to authenticate using the `gcloud` cli that also works with this development workflow, I can make it part of this PR.

Reply:
I've renamed GCP_CREDENTIALS to GOOGLE_APPLICATION_CREDENTIALS to be more consistent. If I understand correctly, it seems like this is the way that things are being done now with the GKEPodOperator, where credentials are securely written into the container and the environment variable is set.